backend/provisioner/noop_provisioner.go

```diff
@@ -0,0 +1,35 @@
+package provisioner
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1beta1/provisioner"
+	"github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1beta1/provisioner/provisionerconnect"
+)
+
+// NoopProvisioner is a provisioner that does nothing
+type NoopProvisioner struct{}
+
+var _ provisionerconnect.ProvisionerPluginServiceClient = (*NoopProvisioner)(nil)
+
+func (d *NoopProvisioner) Ping(context.Context, *connect.Request[ftlv1.PingRequest]) (*connect.Response[ftlv1.PingResponse], error) {
+	return &connect.Response[ftlv1.PingResponse]{}, nil
+}
+
+func (d *NoopProvisioner) Plan(context.Context, *connect.Request[provisioner.PlanRequest]) (*connect.Response[provisioner.PlanResponse], error) {
+	panic("unimplemented")
```

Comment:
Just curious - do we want this to actually panic, or just do nothing?

Reply:
That is just a placeholder until I get to implementing the Planning feature. I thought panicing would work as a better reminder to actually implement it 😄

Reply:
hehe sgtm!